setup.py

```diff
@@ -1,13 +1,19 @@
 # -*- coding: utf-8 -*-
 
-from setuptools import setup
+from setuptools import setup, find_packages
+
+
+def readme():
+    with open('README.rst', encoding='utf8') as f:
```

Comment:
In python 2.x I am getting the following error when running `python setup.py install`:

> Traceback (most recent call last):
>   File "setup.py", line 16, in <module>
>     long_description=readme(),
>   File "setup.py", line 7, in readme
>     with open('README.rst', encoding='utf8') as f:
> TypeError: 'encoding' is an invalid keyword argument for this function

I think using `io.open` would fix the issue. Could you please take a look at it?

http://stackoverflow.com/a/25050323/69537